pom.xml

```diff
@@ -326,24 +326,28 @@
                 </configuration>
             </plugin>
             <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
```

Comment:
Don't we need the groupId?

Reply:
It was already missing it. But I will add it